faker/providers/date_time/__init__.py

```diff
@@ -726,9 +726,9 @@ def time_series(self, start_date='-30d', end_date='now', precision=None, distrib
 
         datapoint = start_date
         while datapoint < end_date:
-            datapoint += precision
             dt = timestamp_to_datetime(datapoint, tzinfo)
             yield (dt, distrib(dt))
+            datapoint += precision
```

Comment:
can we move this one line up, just above the `yield`? mostly a stylistic preference